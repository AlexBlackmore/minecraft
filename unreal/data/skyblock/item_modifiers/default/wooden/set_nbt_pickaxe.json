{
  "function":"set_nbt",
  "tag":"{Rarity:0,Name:\"Wooden Pickaxe\"}"
}
