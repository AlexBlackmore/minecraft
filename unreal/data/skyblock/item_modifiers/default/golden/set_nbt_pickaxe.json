{
  "function":"set_nbt",
  "tag":"{Rarity:0,Name:\"Golden Pickaxe\"}"
}
